diff --git a/core/modules/views_ui/src/ViewEditForm.php b/core/modules/views_ui/src/ViewEditForm.php
index 530a47f..24c1927 100644
--- a/core/modules/views_ui/src/ViewEditForm.php
+++ b/core/modules/views_ui/src/ViewEditForm.php
@@ -509,7 +509,12 @@ public function getDisplayDetails($view, $display) {
     $build['columns']['third'] = array(
       '#type' => 'details',
       '#title' => $this->t('Advanced'),
-      '#theme_wrappers' => array('details'),
+      '#theme_wrappers' => array(
+          'details' => array(),
+          'container' => array(
+          '#attributes' => array('class' => 'details-wrapper'),
+          ),
+      ),
       '#attributes' => array(
         'class' => array(
           'views-display-column',
